Foundation/NSURLSession/URLSessionTask.swift

```diff
@@ -521,7 +521,7 @@ open class URLSessionStreamTask : URLSessionTask {
 }
 
 /* Key in the userInfo dictionary of an NSError received during a failed download. */
-public let URLSessionDownloadTaskResumeData: String = "NSURLSessionDownloadTaskResumeData"
+public let URLSessionDownloadTaskResumeData: String = "URLSessionDownloadTaskResumeData"
```

Comment:
Yes, I was careful to avoid those modifications, but one slipped through the cracks. Thx!